src/sql/azdata.proposed.d.ts

```diff
@@ -680,13 +706,29 @@ declare module 'azdata' {
 
 		export type DialogWidth = 'narrow' | 'medium' | 'wide' | number;
 
+		export type DialogStyle = 'normal' | 'flyout' | 'callout' | 'calloutCompact';
```

Comment:
please add comment for the different styles.

Reply:
callout is ok but calloutCompact doesn't seem to be a proper value here, this is style, calloutCompact is a mix of style and width, it seems to me you can achieve the same by setting the style to callout and setting the width to certain pixels.

Reply:
I removed calloutCompact and revised all conditional logic which was using that name.